pkg/serverless/trace/trace.go

```diff
@@ -67,7 +68,11 @@ const invocationSpanResource = "dd-tracer-serverless-span"
 
 // Load loads the config from a file path
 func (l *LoadConfig) Load() (*config.AgentConfig, error) {
-	return tracecmdconfig.LoadConfigFile(l.Path)
+	c, err := compcorecfg.NewServerlessConfig(l.Path)
+	if err != nil || c == nil {
+		return nil, err
+	}
```

Comment:
Let's make sure `newConfig` just never returns them both as nil, since that's a Go anti-pattern anyway. e.g. either `err` or `c` must be nil, but not both.

Reply:
Yes, good catch.